docs/source/getting-started/setting-up-hypervisor-driver-plugin.adoc

```diff
@@ -75,14 +74,14 @@ $ newgrp libvirt
 ----
 
 [[xhyve-driver-install]]
-== xhyve driver
+=== xhyve driver
 
 Minishift is currently tested against `docker-machine-driver-xhyve` version 0.3.1.
 
 [[homebrew-install]]
-=== Homebrew install
+==== Homebrew install
 
-You can verify the installed version of the xhyve driver on your system before you install.
+You can verify the existing version of the xhyve driver on your system before you install the latest version using:
```

Comment:
`You can verify the existing version of the xhyve driver on your system before you install the latest version using:` to
```
You can verify the existing version of the xhyve driver on your system using:
```
as above line only require if we have mandatory rule to keep the xhyve driver plugin latest. It's user choice or sometime as part of issue where user need to update the xhyve driver.